sessions/computer-setup/is-token-correct/README.md

```diff
@@ -0,0 +1,10 @@
+# Computer Setup
+
+We are verifying that you have done the computer setup correctly.
```

Comment:
We are checking to confirm that your computer setup has been executed correctly.

Upon downloading this folder, if you observe the message "using Github API with token" in your terminal, just as shown in the image below, your computer setup has been executed as intended.

Otherwise, it suggests that you have missed step 8 in the installation guide.